.github/workflows/ciwheel.yml

```diff
@@ -24,38 +24,10 @@ jobs:
         with:
           fetch-depth: 0
           submodules: recursive
-      - name: Update pip
-        run: python -m pip install --upgrade pip
-
-      - name: Build wheels Linux
-        if: ${{ startsWith(matrix.os, 'ubuntu') }}
-        uses: pypa/cibuildwheel@v2.3.0
-        with:
-          output-dir: dist
-        env:
-          CIBW_BEFORE_ALL: yum -y install libxml2-devel
-          CIBW_BEFORE_BUILD: python -m pip install numpy setuptools scikit-build ninja cmake
-          CIBW_BUILD: "cp3*-manylinux_x86_64"
-          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
-          CIBW_ARCHS_LINUX: x86_64
-          CIBW_REPAIR_WHEEL_COMMAND: 'auditwheel repair -w {dest_dir} {wheel} && python /project/scripts/patchwheel.py {dest_dir}'
-          CIBW_TEST_COMMAND: python -m unittest discover -v -s {project}/python
-
-      - name: Build wheels macos
-        if: ${{ startsWith(matrix.os, 'macos') }}
-        uses: pypa/cibuildwheel@v2.3.0
-        with:
-          output-dir: dist
-        env:
-          MACOSX_DEPLOYMENT_TARGET: "10.15" #needed to undo some CIBW settings
-          CIBW_BEFORE_BUILD: python -m pip install numpy setuptools scikit-build ninja cmake
-          CIBW_BUILD: "cp3*-macosx_x86_64"
-          CIBW_ARCHS_MACOS: x86_64 universal2
-          CIBW_TEST_COMMAND: python -m unittest discover -v -s {project}/python
-
-      # this action runs auditwheel automatically with the following args:
-      # https://cibuildwheel.readthedocs.io/en/stable/options/#repair-wheel-command
-
+      - name: Install cibuildwheel
+        run: python3 -m pip install cibuildwheel
+      - name: Build wheels
+        run: python3 -m cibuildwheel --output-dir dist
```

Comment:
We could switch to pypa's Github Action?

Reply:
The reasons for switching from their Action is that this way we're always running their latest. With the Action you need to pick a version... I prefer to catch their updates automatically (and get failure reports early).

Reply:
👍